gke-deploy/deployer/deployer.go

```diff
@@ -26,6 +27,10 @@ const (
 	managedByLabelKey  = "app.kubernetes.io/managed-by"
 
 	managedByLabelValue = "gcp-cloud-build-deploy"
+	//temporary folder pattern used to create tmp folders for the files downloaded from GCS
+	K8sConfigStagingDir = "gke_deployer_temp_"
```

Comment:
deployer -> deploy

Reply:
Do we maybe want to differentiate between the input staging dir and the output staging dir?

Reply:
TempDir() creates a directory with a random string at the end of the name.